NextCenturyCorporation/MCS
# Mcs 1957 windows auto downloader

Adds downlaod support for windows.

The windows addressables and unity exec are in `development` only atm. To test, in `machine_common_sense/_version.py` change the version to 'development'.

To test:

You can then cache addressables:
```
python ./machine_common_sense/scripts/cache_addressables.py
```

Run integration:
```
cd integration_tests
python run_handmade_tests.py
```

You should see it download the `develop` version of the unity executable (MCS-AI2-THOR-Unity-App-vdevelop-{linux || win64 || mac}.zip:). I've tested the auto downloader in windows and linux, but a second set of eyes would be good. Also, can someone make sure nothing is broken in MacOS.

Thanks!
